changelog.txt

```diff
@@ -0,0 +1,4 @@
+feature - Upgraded Ember dependency to 1.9.
+feature - Upgraded Ember Data dependency to 1.0.0-beta.12.
+feature - Update blog example to use Firebase Hosting.
```

Comment:
These last two items don't really belong in the official changelog. Changelog items should only be for changes in the client itself, not for any examples or documentation. I'd take them out.